examples/select_region.cf

```diff
@@ -44,6 +44,7 @@ bundle agent main
                   MATCHING inside the region described by regular expressions
                   matching the beginning and end of the region.",
         create    => "true",
+        #action => warn_or_fix( "warn" ),
```

Comment:
I don't understand, this is only a comment. What is the change here?

Reply:
I have re-visited this when time allowed over a long period, it was just the last change that I was making, all the stuff is squashed now.